app/components/markdown/markdown_image/markdown_image.js

```diff
@@ -268,6 +269,19 @@ export default class MarkdownImage extends React.Component {
                     </TouchableWithFeedback>
                 );
             }
+        } else if (isSvgLink(this.props.source)) {
+            image = (
+                <TouchableWithFeedback
+                    onLongPress={this.handleLinkLongPress}
+                    onPress={this.handlePreviewImage}
+                >
+                    <SvgUri
+                        width='100%'
+                        height='100'
```

Comment:
this seem to be a random value, care to explain the reason to use `100`? what happens if the image height is lower?

Reply:
SVG images in this case scales to fit their container. 100 pixels just seemed like an appropriate value to get a good view of an image on mobile.

Reply:
Mobile View (ios):
<img width="340" alt="Screen Shot 2020-01-26 at 2 41 56 PM" src="https://user-images.githubusercontent.com/30220112/73140730-2b833c80-404a-11ea-91c9-424365b57406.png">

Chrome Web view: 
<img width="816" alt="Screen Shot 2020-01-26 at 2 42 03 PM" src="https://user-images.githubusercontent.com/30220112/73140735-39d15880-404a-11ea-95d8-115611056a91.png">

Note the lack of color on the Trustpilot svg. There are some SVG's that are rendered inconsistently. That might be an issue for a future ticket?